### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by razumovich**

### CODE REVIEW

File: product-service/src/functions/getProductsById.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import productList from '../common/products.json';
+import { ALLOW_CORS_HEADERS } from '../common/config'
+
+export default async (event) => {
+  try {
+    const products = await Promise.resolve(productList);
+    const { productId } = event.pathParameters;
+    const product = products.find(({ id }) => id === productId);
```
Comment (akondratsky) [inline product-service/src/functions/getProductsById.js:8]: It's recommended to split business logic from endpoints. As I remember, it was described in last Q&A session. Also last point of our task mention "try-catch" block in our endpoints. I believe, it's good idea to create service method and wrap it with try..catch.

File: product-service/src/functions/getProductsById.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      body: JSON.stringify(product),
+    } : {
+      statusCode: 404,
+      headers: ALLOW_CORS_HEADERS,
+      body: JSON.stringify({ message: 'Product is not found' }),
+    }
+
+    return response
+  } catch (e) {
+    console.error(e);
```
Comment (akondratsky) [inline product-service/src/functions/getProductsById.js:21]: What endpoint will return if any error? I suggest to send 500 error.

File: product-service/src/functions/getProductsById.js (hunk 1 above)
Comment (razumovich) [reply product-service/src/functions/getProductsById.js]: Done

File: product-service/src/functions/getProductsById.js (hunk 2 above)
Comment (razumovich) [reply product-service/src/functions/getProductsById.js]: Done

**Pull request merged**